cpp/src/parquet/column_reader_benchmark.cc

```diff
@@ -56,19 +58,36 @@ class BenchmarkHelper {
     }
   }
 
-  Int32Reader* ResetReader() {
+  Int32Reader* ResetColumnReader() {
     std::unique_ptr<PageReader> pager;
     pager.reset(new test::MockPageReader(pages_));
     column_reader_ = ColumnReader::Make(descr_.get(), std::move(pager));
     return static_cast<Int32Reader*>(column_reader_.get());
   }
 
+  RecordReader* ResetRecordReader(bool read_dense_for_nullable) {
+    std::unique_ptr<PageReader> pager;
+    pager.reset(new test::MockPageReader(pages_));
+    internal::LevelInfo level_info;
+    level_info.def_level = descr_->max_definition_level();
+    level_info.rep_level = descr_->max_repetition_level();
+    record_reader_ = internal::RecordReader::Make(
+        descr_.get(), level_info, ::arrow::default_memory_pool(),
+        /*read_dictionary=*/false,
+        /*read_dense_for_nullable=*/read_dense_for_nullable);
```

Comment:
nit: the comment is redundant as the argument is self-descriptive

Reply:
Done.